bciers/apps/reporting/src/app/components/additionalInformation/newEntrantInformation/NewEntrantInformationForm.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+
+import React, { useState } from "react";
+import MultiStepFormWithTaskList from "@bciers/components/form/MultiStepFormWithTaskList";
+import { TaskListElement } from "@bciers/components/navigation/reportingTaskList/types";
+import { useRouter } from "next/navigation";
+
+import { actionHandler } from "@bciers/actions";
+import {
+  NewEntrantSchema,
+  NewEntrantUiSchema,
+} from "@reporting/src/data/jsonSchema/newEntrantInformation";
+import { IChangeEvent } from "@rjsf/core";
+
+const baseUrl = "/reports";
+const cancelUrl = "/reports";
+
+interface AdditionalReportingDataProps {
+  versionId: number;
+  initialFormData: { assertion_statement?: boolean };
+}
+
+export default function NewEntrantInformationForm({
+  versionId,
+  initialFormData,
+}: AdditionalReportingDataProps) {
+  const [formData, setFormData] = useState(initialFormData || {});
+  const [submitButtonDisabled, setSubmitButtonDisabled] = useState(
+    !initialFormData.assertion_statement,
+  );
+
+  const router = useRouter();
+  const saveAndContinueUrl = `/reports/${versionId}/compliance-summary`;
+  const taskListElements: TaskListElement[] = [
```

Comment:
We should add this in a `3_additionalInformation.ts` file in `apps/reporting/src/app/components/taskList/` that we can reuse, instead of making it part of this component